puli.json

```diff
@@ -7,7 +7,7 @@
             "class": "Http\\Message\\MessageFactory\\DiactorosMessageFactory",
             "type": "Http\\Message\\MessageFactory",
             "parameters": {
-                "depends": "Zend\\Diactoros\\Request"
+                "depends": "Laminas\\Diactoros\\Request"
```

Comment:
not sure how many people use puli, but i think we should only add the laminas things but not replace zend.

Reply:
I am not using puli and also not sure about its usefullness in general, but the way in understand this would be that adding two dependencies would mean that both (AND) are required and not only one (OR). If this is the case, then we should leave it with laminas here.

Reply:
@Nyholm do you know if we can make puli detect both the old and the new class?

if we can't i opt to configure it with the old class. when the BC layer is there, that should even still work.

Reply:
@Nyholm wdyt?

Reply:
Im honestly not sure either.. :/

Reply:
Ok, then we should probably stick to the old class name...